.gitmodules

```diff
@@ -28,3 +28,6 @@
 [submodule "grpc-server-app-kt-plugin"]
 	path = grpc-server-app-kt-plugin
 	url = https://github.com/stack-spot/grpc-server-app-kt-plugin
+[submodule "openapi-gradle-app-plugin"]
+	path = openapi-gradle-app-plugin
+	url = https://github.com/stack-spot/uxservices-kotlin-openapi-gradle-app-plugin
```

Comment:
Please run `git submodule add https://github.com/stack-spot/uxservices-kotlin-openapi-gradle-app-plugin` to fully configure this.